homework/05LessonFive.js

```diff
@@ -63,15 +63,15 @@ for (forLoopCounter = 0; forLoopCounter > -10; forLoopCounter--) {
 
 // Problem 8:
 // Fill in forLoopArrayResult so that it matches forLoopArray
-var forLoopArrayResult = [];
+var forLoopArrayResult = [0, -1, -2, -3, -4, -5, -6, -7, -8, -9];
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
 // What are loops? Why do we use them?
-//
+// ANSWER: A loop is code written to perform a repetitive task when you know how many times you want to repeat the task.  We use them so we don't have to write lengthy code or do repetive tasks.
```

Comment:
Nice! Do we ever _not_ know how  many times it'll run?
What if I use `Math.rand()` or get input from `prompt`?